src/containers/Secrets/Secrets.js

```diff
@@ -99,6 +111,31 @@ export /* istanbul ignore next */ class Secrets extends Component {
             lowContrast
           />
         )}
+        {createSuccess &&
+          (!deleteSuccess && (
+            <InlineNotification
+              kind="success"
+              title="Success:"
+              subtitle="Secret created successfully"
+              iconDescription="Clear Notification"
+              className="notificationComponent"
+              data-testid="errorNotificationComponent"
+              onCloseButtonClick={this.props.clearNotification}
+              lowContrast
+            />
+          ))}
+        {deleteSuccess && (
+          <InlineNotification
+            kind="success"
+            title="Success:"
+            subtitle="Secret(s) deleted successfully"
+            iconDescription="Clear Notification"
+            className="notificationComponent"
+            data-testid="errorNotificationComponent"
```

Comment:
I did notice that each `Notification` component has the same `data-testeid`, also, asa DRY approach i'd suggest doing something like:

```
{(createSuccess || deleteSuccess) && (
             <InlineNotification
               kind="success"
               title="Success:"
               subtitle={createSuccess ? "Secret created successfully" : "Secret(s) deleted successfully"}
               iconDescription="Clear Notification"
               className="notificationComponent"
               data-testid="successNotificationComponent"
               onCloseButtonClick={this.props.clearNotification}
               lowContrast
             />
           ))}
```

Reply:
The ```data-testid``` is only used in one test, to see if an error occurs, but as you say the create and delete success I don't think need to have this property and so I have removed it from them. As for the format of the notifications, I have attempted to code it . the way you have suggested however I then get a create success notification for both creating and deleting secrets.